core/src/main/scala/caliban/wrappers/ApolloTracing.scala

```diff
@@ -15,26 +15,45 @@ import zio.query.ZQuery
 
 object ApolloTracing {
 
+  private val isEnabledRef = Unsafe.unsafe(implicit u => FiberRef.unsafe.make(true))
```

Comment:
Should it be lazy?

Reply:
In all other places we define it is as `val`. Since objects are loaded lazily on the JVM, I guess it's probably better if we have it as a `val` since it'd be initialised when we first access the object to create the wrapper rather than during execution?

Reply:
Ah right, I thought it was going to be loaded even though we don't use it. Looks fine then.